lib/thesis/notifications.ex

```diff
@@ -37,7 +37,7 @@ defmodule Thesis.Notifications do
       "other" => []
     }
     |> notifications_from_host_app(conn, notifications())
-    |> notifications_regarding_env(Mix.env)
+    |> notifications_regarding_env(Application.get_env(:thesis, :env))
```

Comment:
Should we fall back to `Mix.env` if this doesn't exist? For backwards compatibility.

Reply:
Hi! According to this topic: https://elixirforum.com/t/mix-env-in-production-no-longer-produces-nil-but-crashes-mix-state-genserver/16165/6
we shouldn't user Mix in production environment and since version 1.7.2 Mix is not included in compiled application. 
And if we haven't this variable in config file `Application.get_env` just returns `nil`:
https://hexdocs.pm/elixir/Application.html#get_env/3
So we shouldn't fallback to Mix.env
P.S. before 1.7.2 Mix.env returns nil in production :)